pkg/havener/havener.go

```diff
@@ -86,6 +87,9 @@ type Havener interface {
 
 	PodExec(pod *corev1.Pod, container string, command []string, stdin io.Reader, stdout io.Writer, stderr io.Writer, tty bool) error
 	NodeExec(node corev1.Node, containerImage string, timeoutSeconds int, command []string, stdin io.Reader, stdout io.Writer, stderr io.Writer, tty bool) error
+	ListSecrets(namespaces ...string) ([]*corev1.Secret, error)
+	ListConfigMaps(namespaces ...string) ([]*corev1.ConfigMap, error)
+	ListCRDItems(string) ([]unstructured.Unstructured, error)
```

Comment:
I wonder if we should aim for the full name here: `ListCustomResourceDefinition`. Just for readability.

Reply:
Sure, I can do that